### ISSUE TITLE
Medical image denoising using convolutional neural network: a residual learning approach

### ISSUE THREAD

**Issue by ghuijo** [One-Day-One-Paper/One-Day-One-Paper#242]
* develop medical image denoising model that benefits over small training dataset
* design a deep feed forward convolutional neural network model which directly approximate the noise from a noisy image
* residual learning is used as learning algorithm, and batch normalization is also incorporated to boost model performance
* => denoise the medical image with residual learning approach
* residual learning phase of the model for medical image denoising
  <image src="https://user-images.githubusercontent.com/67627471/180651286-16b3ab5d-a9f2-417c-b182-cd42cdae70fa.png" width=60% height=60%>
* Reconstruction phase
  <image src="https://user-images.githubusercontent.com/67627471/180651298-add7122a-b2a4-422c-99f2-8269b641ad28.png" width=50% height=50%>
* residual function R(x) = S(x) − x
* original function xˆ = R(x) + x
* test accuracy of the MNIST network
  <image src="https://user-images.githubusercontent.com/67627471/180651478-f55f8eff-5018-4695-98cc-93b4f41c9529.png" width=50% height=50%>
* 이대로 구현해봤는데 test 결과가 좋지 않음,,
* 자글자글하게 눈에 띄는 노이즈에만 유의미한 효과가 있는 건가?
